vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -282,17 +296,90 @@ def parse_container_definition(container_def, pod_id)
 
     def parse_container(container, pod_id)
       {
-        :type          => 'ContainerKubernetes',
-        :ems_ref       => "#{pod_id}_#{container.name}_#{container.image}",
-        :name          => container.name,
-        :image         => container.image,
-        :restart_count => container.restartCount,
-        :backing_ref   => container.containerID,
-        :image_ref     => container.imageID
+        :type            => 'ContainerKubernetes',
+        :ems_ref         => "#{pod_id}_#{container.name}_#{container.image}",
+        :name            => container.name,
+        :image           => container.image,
+        :restart_count   => container.restartCount,
+        :backing_ref     => container.containerID,
+        :image_ref       => container.imageID,
+        :container_image => parse_container_image(container.image, container.imageID)
       }
+
       # TODO, state
     end
 
+    def parse_container_image(image, imageID)
+
+      container_image_registry = nil
+      container_image = {
+          :name      => nil,
+          :tag       => nil,
+          :image_ref => imageID
+      }
+
+      slash_count = image.count('/')
+
+      if slash_count > 1
+        registry_index = image.index('/')
+        registry_uri = image[0 .. registry_index - 1].split(':')
+        container_image_registry = {
+            :name => registry_uri[0],
+            :host => registry_uri[0],
+            :port => registry_uri[1],
+            :container_images => []
+        }
+      else
+        registry_index = -1
+      end
+
+      tag_index = image[registry_index + 1 .. -1].rindex(':')
+
+      unless tag_index.nil?
+        container_image.merge!(
+            :name => image[registry_index + 1 .. registry_index + tag_index],
+            :tag =>  image[registry_index + tag_index + 2 .. -1]
+        )
+      else
+        container_image[:name] = image[registry_index + 1 .. -1]
+      end
+
+      host_port = nil
```

Comment:
> Use the default port here: 5000

@simon3z This is a string I use in ```@data_index```
The reason its a string is so I can easily access all the registries or images using 
```Ruby
 data_index.fetch_path(:container_image, :by_ref_and_registry_host_port).try(:values)
```